src/lib.rs

```diff
@@ -1,7 +1,6 @@
 #![allow(clippy::result_large_err)]
```

Comment:
Forbid unsafe in the bot
```suggestion
#![allow(clippy::result_large_err)]
#![forbid(unsafe_code)]
```